core/trino-main/src/test/java/io/trino/operator/join/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -78,10 +80,10 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.openjdk.jmh.annotations.Mode.AverageTime;
-import static org.openjdk.jmh.annotations.Scope.Thread;
 
 @SuppressWarnings("MethodMayBeStatic")
-@State(Thread)
+@State(Scope.Benchmark)
```

Comment:
nit: commit message
`Adjust join benchmark to be reflect reality more` -> `Adjust join benchmark to better reflect reality`